ArduCopter/wscript

```diff
@@ -34,8 +34,15 @@ def build(bld):
         use='mavlink',
     )
 
-    bld.ap_program(
-        program_name='arducopter',
-        program_groups=['bin', 'copter'],
-        use=vehicle + '_libs',
+    frames = (
+        'quad', 'tri', 'hexa', 'y6', 'octa', 'octa-quad', 'heli', 'single',
+        'coax',
     )
+    for frame in frames:
+        frame_config = frame.upper().replace('-', '_') + '_FRAME'
+        bld.ap_program(
+            program_name='arducopter-%s' % frame,
+            program_groups=['bin', 'copter'],
+            use=vehicle + '_libs',
```

Comment:
humn... does this work considering we would actually need a different lib per vehicle since changing the define below changes the compile-time behavior of the libraries?